_tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var StudentsAndGraduates = require( '../../page_objects/page_careers-students-and-graduates.js' );
+
+describe( 'Beta: Careers/Student-and-graduates', function() {
```

Comment:
I got another PR where I'm adding ESLint and fixing up consistency and linting errors like this one ready to go once this is in. :ticket: